lnrpc/routerrpc/router.proto

```diff
@@ -161,6 +161,22 @@ message SendToRouteRequest {
 
     /// Route that should be used to attempt to complete the payment.
     lnrpc.Route route = 2;
+
+    /**
+    If non-zero, the total amount in satoshis being sent as part of a larger
+    multi-path payment. The caller is responsible for ensuring subpayments to
```

Comment:
This comment is a bit confusing, should say something like "the total amount in satoshis of the payment if this is a sub payment of smaller amount"? I don't know what the best way to put this..